.ci/scripts/test_int_cslc_s1.sh

```diff
@@ -52,30 +52,32 @@ aws s3 cp s3://operasds-dev-pge/${PGE_NAME}/validate_cslc.py "$local_validate_cs
 # 0 - pass
 # 1 - failure to execute some part of this script
 # 2 - product validation failure
-overall_status=0
+overall_status=1
```

Comment:
I'm interested to know why you initialized to failure.

Reply:
Looking at this again, it doesn't seem necessary but it made it clear to me that the script would not return a "pass" status unless the PGE execution returned a zero status (where overall_status is now set to 0).

I removed the "set -e" line to allow the script to get through python exceptions I was seeing during the comparison step, and that opened the possibility of getting further in the script after earlier things failed.

I now think a slightly better approach would be to restore the "set -e" and run the comparisons in a way that captures the return values from the comparison step without exiting the script via "set -e" as described here: https://stackoverflow.com/questions/18621990/bash-get-exit-status-of-command-when-set-e-is-active

Reply:
I like this approach you linked, and would like to keep the `set -e` in place